src/server/game/variant1.js

```diff
@@ -439,7 +443,7 @@ const variant1 = (function() {
      * @param {Object} [position] - The starting position of the game, organized by key `{ '1,2': 'queensB' }`, if it's already known. If not provided, it will be calculated.
      * @returns {Object} The gamerules object for the variant.
      */
-    function getGameRulesOfVariant({ Variant, Date = math1.getUTCDateTime() }, position) {
+    function getGameRulesOfVariant({ Variant, Date = math.getUTCDateTime() }, position) {
```

Comment:
Same as above! Follow the same steps to rename all instances of `math` to `math1`

Reply:
Done!